occurrences/models.py

```diff
@@ -519,6 +517,17 @@ def send_event_notifications_to_contact_people(
             )
 
     def approve(self, custom_message=None):
+        group_size = (
```

Comment:
Since now pending enrolment is not count to the seats taken immediately, we need to move the group size validation into `.approve` function